test/nginx_modsec_ingress_ctl_test.go

```diff
@@ -0,0 +1,84 @@
+package integration_tests
```

Comment:
modsec test should have annotations to enable-modsec. And the tests should check for good and bad url. Check existing modsec for reference